lrbd

```diff
@@ -1513,9 +1513,22 @@ class TPGs(object):
         self.portal_index = portal_index
         self.lun_assignment = lun_assignment
 
+        self.disable_auto_add_default_portal()
         self._add()
         Runtime.config['portals'] = self.portal_index.portals
 
+    def disable_auto_add_default_portal(self):
+        """
+        Disable auto portal of 0.0.0.0
+        """
+        proc = Popen(["targetcli", "get", "global", "auto_add_default_portal"],
+                     stdout=PIPE, stderr=PIPE)
+        for line in proc.stdout:
+            results = re.split(r'=', line.decode('utf-8'))
+            if results[1].rstrip() != 'false':
+                cmd = ["targetcli", "set", "global", "auto_add_default_portal=false"]
+                popen(cmd)
+
```

Comment:
Thanks for fixing this Eric. Any particular reason for the status check prior to set? Looks good either way, just curious.